### ISSUE TITLE
Rasa model from API.ai SmallTalk 

### ISSUE THREAD

**Issue by znat** [RasaHQ/rasa_nlu#547]
Hi,
I wanted to share my Friday pm experiment: Porting API.ai's small talk to Rasa.

Testing the model against the training set shows an accuracy of 94%. Not bad for a model with 80+ intents :)
The training set and failed examples are attached [smalltalk.zip](https://github.com/RasaHQ/rasa_nlu/files/1253023/smalltalk.zip), but here are a couple of interesting examples making me wonder about API.ai internals:

```
    {
        "expected": {
            "text": "not bad", 
            "intent": "smalltalk.appraisal.good"
        }, 
        "actual": {
            "intent": "smalltalk.appraisal.bad"
        }
    }, 
    {
        "expected": {
            "text": "not too bad", 
            "intent": "smalltalk.appraisal.good"
        }, 
        "actual": {
            "intent": "smalltalk.appraisal.bad"
        }
    },
{
        "expected": {
            "text": "so nice of you", 
            "intent": "smalltalk.appraisal.thank_you"
        }, 
        "actual": {
            "intent": "smalltalk.appraisal.good"
        }
    }, 

```
The 1st 2 examples behave as if sentiment analysis was done.
The 3rd one is puzzling as a lot of sentences with nice appear in the `smalltalk.appraisal.good` intent. It's the only sentence of this type in `smalltalk.appraisal.thank_you`

The strangest thing is API.ai agent is ready immediately after importing training data (I created an empty agent and imported the small talk dataset). Which makes me wonder if there is any training at all.

**rasa NLU version** (e.g. `0.7.3`): 0.10 a

**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...):spacy_sklearn

**Comment by tmbo** [comment]
The model is probably pretrained (or trained on a large instance which should be quick as well).  Any insights on what we could add?

**Comment by znat** [comment]
I doubt the model is pretrained since I uploaded the intents to a blank agent.
If there is a training, then the resulting model is able to pick up very subtle nuances to recognize not bad as 'good' with only 2 such double negative instances against so many 'bad' occurrences in a true negative context. May be they are using sentiment as a feature, and tagging 'so nice of you' as neutral helps in not getting picked by `smalltalk.appraisal.good` where most instances have a positive sentiment. 

Or may be they just do pattern matching or memorize examples, which might explain why small variations in examples result in random results.

Anyways, this is not the api.ai repo, but it's still interesting to see how results from the same training  set vary between NLU platforms.

**Comment by fakesecret** [comment]
@znat Maybe you can try my preferred pipeline:
"pipeline": ["nlp_mitie", "nlp_spacy", "tokenizer_spacy", "ner_crf", "intent_entity_featurizer_regex", "intent_featurizer_spacy", "intent_classifier_sklearn"]

The example of "so nice of you" still makes sense when expressing "good". But the examples of "good" and "bad" are totally wrong, where we need to teach the model what 'not' means. We may achieve that by adding more examples for "not"
